src/cli/litra-toggle.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
```

Comment:
Can we rename this file to `litra-oversight`? The interface that Oversight uses for scripts is quite specific, so I think it makes sense to name the script in a way that is very explicit about what it's for.